sampleconfig/configtx.yaml

```diff
@@ -258,8 +268,9 @@ Application: &ApplicationDefaults
 Orderer: &OrdererDefaults
 
     # Orderer Type: The orderer implementation to start.
-    # Available types are "solo", "kafka" and "etcdraft".
-    OrdererType: solo
+    # Available types are "solo", "kafka", "etcdraft" and "BFT".
+    # Please note that "solo" and "kafka" are deprecated.
```

Comment:
```
# Available types are "etcdraft" and "BFT".
# Please note that "solo" and "kafka" are no longer supported.
```

Reply:
Changed that, please note that there are profiles using Kafka and Solo in there.

Reply:
Do a text search on the fabric code base and find out which of those solo and kafka profiles are not in use, and delete them. I think there should not be any kafka profile in use. Solo might be found here and there in unit tests, it will take some time to clean that up (in another PR).